### PR TITLE
Update documentation for http probe plugin SLA support

### PR DESCRIPTION
**Pull request by jiangsunan**
Update documentation for http probe plugin SLA support

### CODE REVIEW

**Comment by jamesesilvia** [comment]
- [ ] wait

File: docs/plugin_http_probe.md [hunk 1]
```diff
@@ -275,12 +350,43 @@ Nov 07 03:27:34 my-router.openstacklocal systemd[1]: Started HTTP Monitor Status
...
 ## Release Notes
 
+### Release 1.1.0
+
+**Release Date:** Apr 16, 2024
+
+#### New Features and Improvements
+- **PLUGIN-2300** Implement SLA monitoring per probe
+
+The plugin allows Users to configure the following SLA settings: max-loss, max-jitter, average-rtt. These are considered in probe success criteria. Additionally, these values are available thorugh metrics.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_http_probe.md:388]:
```suggestion
The plugin allows Users to configure the following SLA settings: max-loss, max-jitter, average-rtt. These settings are considered as part of the probe success criteria. Additionally, these values are available through metrics. 
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Comment by jiangsunan** [comment]
@Chr1st0ph3rTurn3r Thanks for the review!!

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/plugin_http_probe.md [hunk 2]
```diff
@@ -56,7 +60,90 @@ config
 exit
 ```
 
-### Service route configuration
+Above configuration runs http probe as follow:
+![Http Probe Timeline](/img/http-probe-timeline.png)
+Based on the configuration, every test runs five probe attempts. Each attempt runs with a two second hard timeout, calculated by the `probe-duration` divided by the `number-of-attempts`. Each probe may fail due to a request failure, timeout reached, or an invalid response code. Service path status is determined at the end of the probe duration and the status will remain until the end of the next test.
```
Comment (agrawalkaushik) [inline docs/plugin_http_probe.md:65]: this feels like its missing a word here

> status will remain (up?/down?) until the end of the next test.

File: docs/plugin_http_probe.md [hunk 3]
```diff
@@ -275,12 +359,45 @@ Nov 07 03:27:34 my-router.openstacklocal systemd[1]: Started HTTP Monitor Status
...
+
+
+Completed in 0.21 seconds
+```
 
 ## Release Notes
 
+### Release 1.2.0
+
+**Release Date:** Aug 19, 2024
```
Comment (agrawalkaushik) [inline docs/plugin_http_probe.md:394]:
```suggestion
**Release Date:** Aug 20, 2024
```

File: docs/plugin_http_probe.md (hunk 2 above)
Comment (agrawalkaushik) [reply docs/plugin_http_probe.md:65]:
```suggestion
Based on the configuration, every test runs five probe attempts. Each attempt runs with a two second hard timeout, calculated by the `probe-duration` divided by the `number-of-attempts`. Each probe may fail due to a request failure, timeout reached, or an invalid response code. Service path status is determined at the end of the probe duration and the current status will remain until the end of the next test.
```

**Review by agrawalkaushik — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**